tests/ember-electron/main.js

```diff
@@ -47,15 +47,3 @@ app.on('ready', function onReady() {
     mainWindow = null;
   });
 });
-
-let cwd = __dirname || resolve(dirname());
```

Comment:
I agree that it doesn't belong in the blueprint, but the whole `tests/` directory of this project is basically an integration test for the project itself. It allows us to run `ember electron:test` and make sure that various things work. So, while we wouldn't want this code in the blueprint because it doesn't belong in applications using this addon, it makes a lot of sense to include it here so that every time we run `ember electron:test` in *this* project we are also verifying that we can access native dependencies.

I don't think it makes sense to try to require that the test code that we have in our dummy test app in this addon exactly matches what's in the blueprint.